src/ProbNumDiffEq.jl

```diff
@@ -60,16 +63,15 @@ _matmul!(
     B::AbstractVecOrMat{T},
 ) where {T<:LinearAlgebra.BlasFloat} = matmul!(C, A, B)
 
-# @reexport using PSDMatrices
-# import PSDMatrices: X_A_Xt
-include("squarerootmatrix.jl")
-const SRMatrix = SquarerootMatrix
+@reexport using PSDMatrices
+import PSDMatrices: X_A_Xt, X_A_Xt!
+# Legacy stuff:
+const SRMatrix = PSDMatrix
```

Comment:
All usages of `SRMatrix` should be replaced with `PSDMatrix` before merging this PR